### PR TITLE
install.sh: Cleanup of quotes

### PR DESCRIPTION
**Pull request by joakimr-axis**
Where no variable expansion is done, " is replaced with '.
Copying content of one variable to another variable never needs quotes.
Some calls to echo and fatal need no quotes at all, but have been left
with ' to make it easier to see where the string begins and ends.
The fatal function now also sends output to stderr.

Change-Id: I2504707e5c550bc498e7663cedf58c224cd0a15b
Signed-off-by: Joakim Roubert <joakimr@axis.com>

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -71,11 +71,11 @@ GITHUB_URL=https://github.com/rancher/k3s/releases
 # --- helper functions for logs ---
 info()
 {
-    echo "[INFO] " "$@"
+    echo '[INFO] ' "$@"
 }
 fatal()
 {
-    echo "[ERROR] " "$@"
+    echo '[ERROR] ' "$@" 2>&1
```
Comment (erikwilson) [inline install.sh]: Should it be `>&2`?

Comment (joakimr-axis) [reply install.sh]: It should, my bad. Thank you! Will fix.

Comment (joakimr-axis) [reply install.sh]: It should, my bad. Thank you! Will fix.

File: install.sh [hunk 2]
```diff
@@ -338,7 +338,7 @@ setup_binary() {
     if command -v getenforce > /dev/null 2>&1; then
         if [ "Disabled" != $(getenforce) ]; then
             if command -v semanage > /dev/null 2>&1; then
-                info "SELinux is enabled, setting permissions"
+                info 'SeLinux is enabled, setting permissions'
```
Comment (yamt) [inline install.sh]: i suspect SELinux is more correct.
https://github.com/SELinuxProject

**Comment by joakimr-axis** [comment]
It seems there are some hickups with the amd64 test pipeline:

```
E1008 05:52:50.222703 1 node.go:124] Failed to retrieve node info: nodes "b31d543f8bcf" not found

E1008 05:52:51.258245 1 core.go:201] failed to start cloud node lifecycle controller: no cloud provider provided

W1008 05:53:02.715802 1 actual_state_of_world.go:506] Failed to update statusUpdateNeeded field in actual state of world: Failed to set statusUpdateNeeded to needed true, because nodeName="b31d543f8bcf" does not exist
```

(Patch has been verified on armv7 device here)

**Pull request merged**